src/org/omegat/filters2/IParseCallback.java

```diff
@@ -55,24 +55,46 @@ public interface IParseCallback {
      * @param filter
      *            filter which produces entry
      * @param protectedParts
-     *            (since 3.0.6) protected parts
+     *            protected parts
      */
     void addEntryWithProperties(String id, String source, String translation, boolean isFuzzy, String[] props,
```

Comment:
A method `addEntryWIthProperties` has 8 parameters, that breaks a checkstyle rule.  Exceeding a number of argument is a signature of design error.